src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,5 +1,15 @@
 import React from 'react';
+import { User } from '../../types/User';
 
-export const UserInfo: React.FC = () => (
-  <>Put the user here</>
+type Props = {
+  user: User
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => (
+  <a
+    className="UserInfo"
+    href={`mailto:${user.email}`}
```

Comment:
Here is the same about destructuring as in previous work

Reply:
Let's use object destructuring here